qiskit/transpiler/preset_passmanagers/level2.py

```diff
@@ -19,50 +19,26 @@
 from qiskit.transpiler.passmanager_config import PassManagerConfig
 from qiskit.transpiler.timing_constraints import TimingConstraints
 from qiskit.transpiler.passmanager import PassManager
+from qiskit.transpiler.passmanager import StagedPassManager
 
-from qiskit.transpiler.passes import Unroller
-from qiskit.transpiler.passes import BasisTranslator
-from qiskit.transpiler.passes import UnrollCustomDefinitions
-from qiskit.transpiler.passes import Unroll3qOrMore
-from qiskit.transpiler.passes import CheckMap
-from qiskit.transpiler.passes import GateDirection
 from qiskit.transpiler.passes import SetLayout
 from qiskit.transpiler.passes import VF2Layout
-from qiskit.transpiler.passes import VF2PostLayout
 from qiskit.transpiler.passes import TrivialLayout
 from qiskit.transpiler.passes import DenseLayout
 from qiskit.transpiler.passes import NoiseAdaptiveLayout
 from qiskit.transpiler.passes import SabreLayout
-from qiskit.transpiler.passes import BarrierBeforeFinalMeasurements
 from qiskit.transpiler.passes import BasicSwap
 from qiskit.transpiler.passes import LookaheadSwap
 from qiskit.transpiler.passes import StochasticSwap
 from qiskit.transpiler.passes import SabreSwap
-from qiskit.transpiler.passes import FullAncillaAllocation
-from qiskit.transpiler.passes import EnlargeWithAncilla
 from qiskit.transpiler.passes import FixedPoint
 from qiskit.transpiler.passes import Depth
 from qiskit.transpiler.passes import Size
-from qiskit.transpiler.passes import RemoveResetInZeroState
 from qiskit.transpiler.passes import Optimize1qGatesDecomposition
 from qiskit.transpiler.passes import CommutativeCancellation
-from qiskit.transpiler.passes import ApplyLayout
-from qiskit.transpiler.passes import CheckGateDirection
-from qiskit.transpiler.passes import Collect2qBlocks
-from qiskit.transpiler.passes import ConsolidateBlocks
-from qiskit.transpiler.passes import UnitarySynthesis
-from qiskit.transpiler.passes import TimeUnitConversion
-from qiskit.transpiler.passes import ALAPScheduleAnalysis
-from qiskit.transpiler.passes import ASAPScheduleAnalysis
-from qiskit.transpiler.passes import ConstrainedReschedule
-from qiskit.transpiler.passes import InstructionDurationCheck
-from qiskit.transpiler.passes import ValidatePulseGates
-from qiskit.transpiler.passes import PulseGates
-from qiskit.transpiler.passes import PadDelay
 from qiskit.transpiler.passes import Error
-from qiskit.transpiler.passes import ContainsInstruction
+from qiskit.transpiler.preset_passmanagers import common
 from qiskit.transpiler.passes.layout.vf2_layout import VF2LayoutStopReason
-from qiskit.transpiler.passes.layout.vf2_post_layout import VF2PostLayoutStopReason
 
 from qiskit.transpiler import TranspilerError
 
```

Comment:
In line 46, 
```
level_2_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassManager:
```

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/6403/commits/a2472f29b01d19a8ee55118d5533e507d6cda871